messages/transaction.go

```diff
@@ -26,3 +28,8 @@ const (
 	// because it will exceed the account daily debit limit.
 	DailyDebitLimitExceeded DebitFailureReason = "daily debit limit exceeded"
 )
+
+// DailyDebitLimitDate formats a time value as a daily debit limit date.
```

Comment:
```suggestion
// DailyDebitLimitDate returns the date of a transaction for the purposes of
// checking daily debit limits.
//
// It normalizes the date to the UTC timezone so that regardless of which
// timezone is used to schedule the transaction two equivalent times always
// contribute towards the same day's limit.
```

Reply:
I forgot to add docs about that UTC conversion.